lib/scroll/sync_scroll_controller_manager.dart

```diff
@@ -81,7 +83,19 @@ class SyncScrollControllerManager {
   void _syncLeftListViewScrollVontroller(
       ScrollController _scrollingController, ScrollController controller) {
     if (_refreshController?.headerStatus != null) {
+      //log(_refreshController!.headerStatus.toString());
+
       switch (_refreshController?.headerStatus) {
+        case RefreshStatus.canRefresh:
+          {
+            //log(_scrollingController.offset.toString() + ", " + _refreshIndicatorHeight.toString());
```

Comment:
Thanks @yingshaoxo ! I have tested web, android and ios which are all good. would you mind remove the log comment and the `import 'dart:developer';` line. then I will accept the PR and later release a new version.😉

Reply:
Hi, MyaLau-CbL. I just did what you asked. It's my pleasure to make a contribution.